argcomplete/shell_integration.py

```diff
@@ -74,7 +74,7 @@ def shellcode(executables, use_defaults=True, shell='bash', complete_arguments=N
     '''
 
     if complete_arguments is None:
-        complete_options = '-o nospace -o default' if use_defaults else '-o nospace'
+        complete_options = '-o nospace -o default -o bashdefault' if use_defaults else '-o nospace'
```

Comment:
We should also include `-o bashdefault` when `use_defaults` is False to match the behaviour of the global completion script.

Reply:
Nice catch. Added as requested.